apps/samourai-dojo/nginx/connect/index.js

```diff
@@ -0,0 +1,30 @@
+var baseRoute = "";
+
+if(bitcoinNetwork == "testnet") {
```

Comment:
Hey I think you've inversed your condition, no?

Reply:
Yep 😁